src/nvme/linux.h

```diff
@@ -41,8 +45,8 @@ enum nvme_telemetry_da {
  * nvme_get_ctrl_telemetry() -
  * @fd:	   File descriptor of nvme device
  * @rae:   Retain asynchronous events
- * @log:   On success, set to the value of the allocated and retreived log.
- * @da:    log page data area, valid values: 1, 2, 3, and 4
+ * @log:   On success, set to the value of the allocated and retrieved log.
+ * @da:    log page data area, valid values: enum nvme_telemetry_da
```

Comment:
kernel-doc needs the '&' for the referencing to work, e.g.

`* @da:    log page data area, valid values: &enum nvme_telemetry_da`

Reply:
@igaw I fixed the kernel-doc warnings and tried to add additional comments to my commit messages.  I was able to update a couple of them but it appears all the commits from merging with the latest master branch changes got included and listed in the pull request.  I'm not sure how to squash them - without screwing it up more.  If you have some advice on how to do that, let me know.   Thanks